homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -68,12 +63,16 @@ async def async_step_import(self, import_info: dict[str, Any]) -> FlowResult:
             _LOGGER.exception(exception)
             return self.async_abort(reason="unknown")
 
-        await self.async_set_unique_id(self._unique_id, raise_on_progress=False)
+        await self.async_set_unique_id(unique_id, raise_on_progress=False)
         self._abort_if_unique_id_configured()
 
-        self._name = import_info[CONF_NAME] or "Radio"
-
-        return await self._create_entry(pin=import_info[CONF_PIN])
+        return self.async_create_entry(
+            title=import_info[CONF_NAME] or "Radio",
+            data={
+                CONF_WEBFSAPI_URL: self._webfsapi_url,
```

Comment:
Inside the import flow, you don't need to set `self._webfsapi_url`
```suggestion
                CONF_WEBFSAPI_URL: webfsapi_url,
```